### PR TITLE
add in user yaml

### PR DESCRIPTION
**Pull request by paulmurdoch19**
also changes some default amanuensis config values and changes portal timeout to 60 seconds, I had some issues with it not starting in time

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -37,13 +37,1049 @@ elasticsearch:
...
+
+      # policies automatically given to authenticated users (in addition to their other
+      # policies)
+      all_users_policies: ['open_data_reader', 'authn_open_access']
+
+      user_project_to_resource:
+        QA: /programs/QA
+        DEV: /programs/DEV
+        test: /programs/QA/projects/test
+        jenkins: /programs/jnkns/projects/jenkins
```
Comment (grugna) [inline values.yaml]: Can you add all the ones from our version of user.yaml to this one? [Here](https://github.com/chicagopcdc/configuration-files/blob/master/cloud/portal-dev.pedscommons.org/configs/user-dev.yaml) for reference.

**Comment by grugna** [comment]
Beside that little change feel free to merge.

File: values.yaml (hunk 1 above)
Comment (paulmurdoch19) [reply values.yaml]: Is there one that I missed? all the values from the config files repo should be in there except for the users jooholee@uchicago.edu, bkang.dev@gmail.com, and slv@uchicago.edu.

Comment (grugna) [reply values.yaml]: that one is the one from CTDS, so the amanuensis resource, policy, role etc etc are missing, and a couple other like analysis, data_admin etc etc

File: values.yaml [hunk 2]
```diff
@@ -37,13 +37,1049 @@ elasticsearch:
...
+        #   resource_paths: ['/']
+
+        # argo
+        - id: argo
+          description: be able to use argo
+          resource_paths: [/argo]
+          role_ids: [argo_user]
+        
+        #PCDC specific
+        - id: 'services.amanuensis-admin'
```
Comment (paulmurdoch19) [inline values.yaml]: sorry they are kind of buried since ctds has so many. should i remove the ctds ones?

**Comment by grugna** [comment]
No, you can keep them all. No need to apologize, I know which one they are because I made them way back, I don't think there is any documentation on it. I should write a new page in the Confluence.

**Comment by paulmurdoch19** [comment]
ok sounds good! is it good to merge?

**Comment by grugna** [comment]
I don't see the missing resource etc etc, did you forget to push the commit?

**Comment by paulmurdoch19** [comment]
I think I am confused all the values from [here](https://github.com/chicagopcdc/configuration-files/blob/master/cloud/portal-dev.pedscommons.org/configs/user-dev.yaml) are in the values.yaml file

**Comment by paulmurdoch19** [comment]
Note: this is not actually ready anymore, the user yaml values are not compatible with the newest version of fence

**Pull request closed without merging**